repo/packages/M/marathon-lb/19/marathon.json.mustache

```diff
@@ -9,7 +9,15 @@
     "docker": {
       "image": "{{resource.assets.container.docker.marathon-lb-docker}}",
       "network": "HOST",
-      "privileged": true
+      "privileged": true,
+      "parameters": [
```

Comment:
I'm over the moon @nlsun you've replied!  I'd love to help out and add more to the config.json, but I'm not clear on how that works or where to inject the explanation of this block.

I scanned over the config.json and looked at/for the jvm settings and other things and it wasn't clear.

Can you slap me toward a better example?

Also, if this lands, I'm MORE than happy to run through other packages as well and add this logic once the maintainers are happy with what I'm providing.